pkg/security/util.go

```diff
@@ -2,6 +2,7 @@ package security
 
 const (
 	UIDRangeAnnotation     = "openshift.io/sa.scc.uid-range"
+	FSGroupAnnotation      = "openshift.io/sa.scc.fsgroup"
```

Comment:
On second thought... maybe what you have will work.  For the case I just described, you could use the first group as the allocated one, and put the allowed supplemental groups for shared storage after that in the list.